test/Generator/PropertyGeneratorTest.php

```diff
@@ -423,7 +423,7 @@ public function testFromReflectionReadonlyProperty(): void
         $generator = PropertyGenerator::fromReflection($reflectionProperty);
         $code      = $generator->generate();
 
-        self::assertSame('    public readonly string $readonly;', $code);
+        self::assertSame('    public readonly $readonly;', $code);
```

Comment:
@allansun readonly is required to have type

Reply:
@samsonasik it was not my test. I removed the type as it should be (check code history, I added it there, which you objected in your earlier review). 

For BC / regression purpose, the result is expected to be without type hint

Reply:
ok 👍 , for note: separate PR is required to solve it as it will produce fatal error https://3v4l.org/n9DA4

Reply:
Noooope, I don't think so... 

First of all, I didn't do the readonly part, it was already merged into the branch before I started working on it. So presumably all existing code should have passed all the tests.

Secondly please refer to comment by  @Ocramius https://discourse.laminas.dev/t/rfc-laminas-laminas-code-add-type-hinting-option-in-propertygenerator/2652/5 I think he's making a valid point that it should depend on the developer to choose which version of PHP the generated code should be compatible with.

Thirdly I can't see why the test would fail since it has /** @requires PHP >= 8.1 */ mark on it

https://github.com/allansun/laminas-code/blob/02b822faec472398edff890077b583922edfd264/test/Generator/PropertyGeneratorTest.php#L413-L427

Reply:
@allansun my note points is not must be you to implement typed readonly, can be other contributor to implement it :)